.env.example

```diff
@@ -1,3 +1,11 @@
+# Random generation ===========================================================
+# If you change this, check the collision probability https://zelark.github.io/nano-id-cc/
+NORAY_RANDOM_ID_LENGTH=21 # For 10 id/hour, 15 trillion years
```

Comment:
Please use the following config keys to make both IDs configurable:
- `NORAY_OID_LENGTH`
- `NORAY_OID_CHARSET`
- `NORAY_PID_LENGTH`
- `NORAY_PID_CHARSET`